packages/firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.8.1+2
+
+* Update README.
+* Update the example app. The example app now have two separate pages includes a `Registration` page and a `SignIn/SignOut` page. The SignIn methods included in the `SignIn/SignOut` page are:
```

Comment:
There are some English language issues with this changelog entry. It might be sufficient to just say "Updated the example app with separate pages for registration and sign-in."